docs/advanced.rst

```diff
@@ -650,3 +650,12 @@ The ``-s`` is necessary because the CherryPy class also wraps stdin and stdout.
    If you want to really unit test your CherryPy application, meaning without
    having to start a server, you may want to have a look at
    this `recipe <https://bitbucket.org/Lawouach/cherrypy-recipes/src/tip/testing/unit/serverless/>`_.
+
+.. note::
+
+   The :class:`helper class <cherrypy.test.helper.CPWebCase>` derives from 
+   ``unittest.TestCase`` class. For this reason, running from  `py.test <http://pytest.org/latest/>`_,
```

Comment:
pytest is officially spelled w/o dot + the dot-style invocation has been deprecated for years... + the link is broken. So please apply this:
```suggestion
   :class:`unittest.TestCase` class. For this reason, running from
   `pytest <https://pytest.org/>`_,
```
(You should see a `Commit suggestion` button below)